nilearn/plotting/tests/test_displays.py

```diff
@@ -72,3 +76,20 @@ def test_user_given_cmap_with_colorbar():
     # Test with cmap given as a string
     oslicer.add_overlay(img, cmap='Paired', colorbar=True)
     oslicer.close()
+
+
+def test_data_complete_mask():
+    """This special case test is due to recent matplotlib 2.1.0.
+
+    When the data is completely masked, then we have plotting issues
+    See similar issue #9280 reported in matplotlib. This function
+    tests the patch added for this particular issue.
+    """
+    if LooseVersion(matplotlib.__version__) == LooseVersion("2.1.0"):
```

Comment:
I think that we should test always, independent of the matplotlib version: the code below should always work.